powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/steps/CreateTokenStep.java

```diff
@@ -148,11 +183,12 @@ public JSONObject execute(JsonStepLogger stepLogger, Map<String, Object> context
             Map<String, String> headers = new HashMap<>();
             headers.put("Accept", "application/json");
             headers.put("Content-Type", "application/json");
-            headers.put(PowerAuthSignatureHttpHeader.HEADER_NAME, httpAuhtorizationHeader);
+            headers.put(PowerAuthSignatureHttpHeader.HEADER_NAME, httpAuthorizationHeader);
+            headers.put(PowerAuthEncryptionHttpHeader.HEADER_NAME, httpEncryptionHeader);
             headers.putAll(model.getHeaders());
 
             if (stepLogger != null) {
-                stepLogger.writeServerCall(uri, "POST", requestObject, headers);
+                stepLogger.writeServerCall(uri, "POST", request.getRequestObject(), headers);
```

Comment:
We try to log the full request object, as is...

Reply:
Ok, I will change it.

Reply:
I looked into the other usages and we usually log the request object, so I will unify all logging.